US County Data - pandas.ipynb

```diff
@@ -7,7 +7,8 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    top_counties.head()
</pre><p>You should really be able to do this without all of this, you should be able to write a filter for the top10 counties and set that equal to a new dataframe.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CMU-MS-DAS-Modern-Programming-Mini/ClassAssignmentNo4/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>